app/gateway/traffic-control/proxy.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Proxying with {{site.base_gateway}}
+
+description: "Proxying is when {{site.base_gateway}} matches an HTTP request with a [registered route](/gateway/entities/route/) and forwards the request."
+
+content_type: reference
+layout: reference
+
+products:
+  - gateway
+
+related_resources:
+  - text: Route entity
+    url: /gateway/entities/route/
+  - text: Router Expressions language
+    url: /gateway/routing/expressions/
+  - text: Expressions repository
+    url: https://github.com/Kong/atc-router
+
+breadcrumbs:
+  - /gateway/
+  - /gateway/traffic-control/
```

Comment:
It's failing because this page doesn't exist `/gateway/traffic-control/` , I'll improve the error message

Reply:
Thanks! I fixed it now